src/service/dashboard/defaultDashboard.flowgraph

```diff
@@ -14,7 +14,7 @@ blocks:
   - name: remote source 1
     id: opendigitizer::RemoteSource
     parameters:
-      remote_uri: http://localhost:8080/GnuRadio/Acquisition?channelNameFilter=test
+      remote_uri: https://localhost:8080/GnuRadio/Acquisition?channelNameFilter=test
```

Comment:
This is now the only part, which still references the host-name of the service. I see that it's not as simple as with the other "localhost" occurrences to update since it is not in c++ code but in a flowgraph yaml file. Maybe this could be either a relative path or contain some token that will be substituted when loading the flowgraph?
Later this will be less important since the flowgraph will only contain the signal name to look up.

Reply:
I think that's ok for now, ideally we'll deploy useful dashboards on a per-instance level, so we'll know the hostname and for testing localhost (which can then be updated by the user to test other models also seems feasible). And again, DNS will make this mostly obsolete so the current state seems acceptable if there is no easy way to make it more generic.